zcash_test_vectors/orchard/key_components.py

```diff
@@ -64,15 +63,15 @@ def __init__(self, chaincode, data):
     def master(cls, S):
         digest = blake2b(person=b'ZcashIP32Orchard')
         digest.update(S)
-        I = digest.digest()
+        I   = digest.digest()
         I_L = I[:32]
         I_R = I[32:]
         return cls(I_R, I_L)
 
     def child(self, i):
         assert 0x80000000 <= i and i <= 0xFFFFFFFF
 
-        I = prf_expand(self.chaincode, b'\x81' + self.data + i2leosp(32, i))
+        I   = prf_expand(self.chaincode, b'\x81' + self.data + i2leosp(32, i))
```

Comment:
Is this spacing method is compatible with the rest of the package?

Reply:
This is how the original orchard file from upstream spaces it.